fnet/cli/predict.py

```diff
@@ -93,6 +93,7 @@ def item_from_dataset(
     else:
         item = dataset[idx]
     target = None
+    
```

Comment:
Why?

Reply:
readability

Reply:
I think this falls into the category of opinion, and I object to changes to files that have no affect on functionality. As a reminder, users can always run `fnet init` and customize the training/predict scripts as they desire.